### ISSUE TITLE
Issue with Rendering Styled Controls and GlobalUseSkia= true

### ISSUE THREAD

**Issue by TheOriginalBytePlayer** [skia4delphi/skia4delphi#217]
See Attached Project Sample 
[FMXStyleRenderBug.zip](https://github.com/skia4delphi/skia4delphi/files/11129359/FMXStyleRenderBug.zip)

With GlobalUseSkia = True you'll see that the progress bar never shows its bar, AND the track bar's track is empty in addition to it having an empty, non-filled circle for the thumb.
![image](https://user-images.githubusercontent.com/18058224/229287048-e2299891-216d-42be-8e21-12c51d991a1a.png)

With GlobalUseSkia =FALSE, it displays as expected, both the progress bar AND Track bar's thumb and track are correct.
![image](https://user-images.githubusercontent.com/18058224/229287144-62cd795b-a41d-48c4-819d-28d4724f041f.png)

My guess is that this has something to do with gradient fills in the style sheets...

**Comment by MyBuzzTechnologies** [comment]
I've just noticed this on a TRectangle with gradient fill since updating to Skia4Delphi 5.0.0.
This is on an FMX project and affects Windows, iOS and Android builds.

Reproduce: 
1. Create a new Multi Device project (FMX)
2. Add a TRectangle. Set Fill.Kind to Gradient
3. Run it and it looks ok
4. Right-click the project, choose Enable Skip
5. Run the project again. The rectangle now appears empty

100% reproducible on all platforms.
Could you please look into this ASAP as it's a big issue for apps like ours that use gradients which no longer show at all.

**Comment by checkdigits** [comment]
Thanks for letting us know. I'll see if this can get looked at ASAP. I've raised a message in the Telegram group too although here in GitHub is the best place for bugs.

**Comment by alexdgs** [comment]
Well, not sure if this will work for anyone else but I found the following workaround for a FMX project:
- Copy file `Skia.FMX.Graphics.pas` from Skia installation to Delphi project.
- Locate procedure `TSkCanvasCustom.BeginPaint` (line 1407), and at the very beginning change `FPaint.Alpha := 0` to `FPaint.Alpha := 255`.

After compilation gradients are painted correctly. The project I tested this is very simple so I'm unaware of any side effects this could have on more complex scenarios.
